examples/e2e/README.md

```diff
@@ -2,32 +2,78 @@
 
 This example provides a simple way to spin up an end to end deployment of package-feeds and the package-analysis sub projects, allowing for easy demonstration of behaviour of the system as a whole.
 
-## Obtaining the necessary images
-
-Images can be pulled from the relevant repositories however some require credentials (github for package-feeds).
-To build the necessary images yourself for the docker-compose, you can do the following:
+## Running
 
+```shell
+$ cd examples/e2e # must be run from the e2e folder
+$ docker-compose up -d
+$ curl localhost:8080
 ```
-# In package-analysis
-cd build
-./build_docker.sh
 
-# In package-feeds
-docker build . -t docker.pkg.github.com/ossf/package-feeds/packagefeeds:latest
-```
+Requesting `localhost:8080` will trigger package-feeds to poll it's feeds and send the packages downstream to package-analysis.
 
-## Running the example
+You may need to run `curl` again if package-feeds is not yet running.
 
-```
-docker-compose up -d
-curl localhost:8080
-```
+## Analysis Output
+
+Output can be found at http://localhost:9000/minio/package-analysis,
+using the following credentials for authentication:
 
-Curling `localhost:8080` will trigger package-feeds to poll it's feeds and send the packages downstream to package-analysis. Outputs can be found at http://localhost:9000/minio/package-analysis,
-using the credentials minio:minio123 for authentication.
+- username: `minio`
+- password: `minio123`
+
+## Logs Access
 
 `docker-compose logs -f feeds` will provide information on the packages which have been send downstream.
 
 `docker-compose logs -f scheduler` will provide information on the packages which have been received and proxied onto the analysis workers.
 
 `docker-compose logs -f analysis` provides too much stdout to be useful, outputs are sent to minio as mentioned above.
+
+## Additional Notes
+
+### Limitations
+
+- Locally built sandbox images are currently ignored.
+
+### Feeds does not start (missing config)
+
+This can happen if `./config` is not world-readable. You will see the error message `open /config/feeds.yml: permission denied` in the feeds logs.
+
+To fix simply run:
+
+```shell
+$ chmod ugo+rx ./config
+$ chmod ugo+r ./config/feeds.yml
+```
+
+### Sandbox container is not starting (cgroups v2)
+
+If the `analysis` logs show failures when trying to start the sandbox container, your machine may be configured to us cgroups v2.
+
+To work with cgroups v2 you will need to:
+
+1. add/edit `/etc/docker/daemon.json` and the following:
+
+```json
+{
+    "default-cgroupns-mode": "host"
+}
+```
+
+2. restart dockerd (if it is running)
```

Comment:
Maybe a silly question, but how do I restart dockerd

Reply:
On a system running systemd: `systemctl restart docker.service`

Reply:
Added to the doc.